### ISSUE TITLE
CO2 data are not read in from file unlike every other input when not passed through memory

### ISSUE THREAD

**Issue by dschlaep** [Burke-Lauenroth-Lab/rSOILWAT2#76]
`rSOILWAT2` can initiate a `SOILWAT2` simulation run in one of two ways:
- pass all input data through memory: this is done through the C function `start` which calls `SW_CTL_init_model` which calls `_read_inputs`. `_read_inputs` if compiled for `rSOILWAT2` takes the inputs from the S4 classes if `useFiles` is FALSE.

- read all input data from disk file -- emulate `SOILWAT2` standalone behavior -- or just read in such files for later manipulation
* If the C function `start` is called without input object but with a directory pointing to the input files, then `_read_inputs` will read those disk files -- however, currently it does not read `carbon.in`.
* If the R function `sw_exec` is called without input object -- or if the input object is created previously by a call to the R function `sw_inputDataFromFiles`, then the C function `onGetInputDataFromFiles` will read the disk files -- however, currently it does not read `carbon.in`.

In other words, `rSOILWAT2` lacks the ability to obtain information from a `carbon.in` file -- in contrast to all the other input files.

**Comment by dschlaep** [comment]
This starts now to work, but the flags are always set to 0L/false when read from files (even when the values on file are 1L/true).

**Comment by dschlaep** [comment]
Previous comment upgraded to separate issue #77
